testing/src/uploadResult.ts

```diff
@@ -0,0 +1,59 @@
+import axios, { AxiosResponse } from "axios";
+import FormData from "form-data";
+import fs from "fs";
+
+const TQ_USERNAME = process.env.TQ_USER_NAME;
+const TQ_PASSWORD = process.env.TQ_PASSWORD;
+const TQ_SECRET = process.env.TQ_SECRET;
+const TQ_RUN_ID = process.env.TQ_RUN_ID; //( default id is 26606. Will need to change once we have integrated TQ account for the team)
+
+const getAuthToken = async (): Promise<AxiosResponse<any>> => {
```

Comment:
Please keep the method declaration consistent. Please use here the same style used for `async function uploadResultFile()`.